src/isa/riscv64/instr/rvv/vcompute_impl.c

```diff
@@ -95,6 +95,14 @@ static inline void update_vcsr() {
 }
 
 void arthimetic_instr(int opcode, int is_signed, int widening, int narrow, int dest_mask, Decode *s) {
+  if(vstart->val >= vl->val) {
```

Comment:
It seems the duplicated code can be wrapped as a function, which is shared by different functions